projects/systelab-translate/README.md

```diff
@@ -8,6 +8,19 @@ Library with I18N tools to speed up our Angular developments
 npm install systelab-translate --save
 ```
 
+## Importing the library 
+
+These lines should be added in app.module file
+```javascript 
+  // required var declaration
+  let _translationFilesUrl = null  
```

Comment:
Do not use _ prefix. for variables. see Angular Code Conventions.
Assign to something.